src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -313,7 +314,7 @@ bool Lowering::IsContainableUnaryOrBinaryOp(GenTree* parentNode, GenTree* childN
             return false;
         }
 
-        const ssize_t wrapAmount = (static_cast<ssize_t>(genTypeSize(parentNode)) * BITS_PER_BYTE);
+        const ssize_t wrapAmount = (static_cast<ssize_t>(genTypeSize(rotateOpNode)) * BITS_PER_BYTE);
```

Comment:
Actually shouldn't this be `childNode`? Can `rotateOpNode` be a small typed local?

Reply:
Correct! x64 also uses `childNode` for it